frontend/test/metabase/scenarios/admin/permissions/sandboxes.cy.spec.js

```diff
@@ -529,7 +528,7 @@ describeWithToken("formatting > sandboxes", () => {
             },
           }).then(({ body: { id: CARD_ID } }) => {
             test === "workaround"
-              ? runAndSaveQuestion({ question: CARD_ID, sandboxValue: "1" })
+              ? visitQuestion({ question: CARD_ID, sandboxValue: "1" })
               : null;
```

Comment:
Might be easier to understand if instead of the ternary we use either

~~~js
if (test === "workaround") {
  visitQuestion({ question: CARD_ID, sandboxValue: "1" })
}
~~~

or

~~~js
  test === "workaround" && visitQuestion({ question: CARD_ID, sandboxValue: "1" })
~~~

Reply:
true. `if`s are more typical here so i'll just switch it to that.

Reply:
This was totally my mess. Thanks for fixing it.

Reply:
Please take a look at [this](https://github.com/metabase/metabase/pull/16579#pullrequestreview-684199079) comment.

While this refactoring is still nice and desirable, it has literary nothing to do with this issue. I missed that initially.
I'll deal with these ternaries separately, but the logic of that test mustn't be changed and everything related to it must be reverted. in this PR.